app/src/lib/stores/app-store.ts

```diff
@@ -3153,4 +3151,69 @@ export class AppStore {
 
     return gitStore.addUpstreamRemoteIfNeeded()
   }
+
+  private forkPullRequestRemoteName(remoteName: string) {
```

Comment:
This can be a function outside the class as it doesn't depend on any internal state